src/nupic/ntypes/Dimensions.hpp

```diff
@@ -37,11 +37,12 @@
 #include <nupic/types/Types.hpp>
 #include <nupic/utils/Log.hpp>
 #include <nupic/types/Serializable.hpp>
+#include <cereal/types/vector.hpp>
```

Comment:
please rm this cereal specific include, all needed (can we make something as `#include <cereal/cereal.hpp>` to include all?) includes should be within Serializable. The nupic classes should only include and implement Serializable.hpp

Reply:
>  includes should be within Serializable. 

It was already in Serializable...just forgot to remove it from here.